### DISCUSSION TITLE
Install PSR4, "ERROR: the "PSR4" coding standard is not installed. "

### DISCUSSION THREAD

**Discussion by GeoffreyOlivier** [phpro/grumphp#952] [Q&A]
| Q               | A
| --------------- | ---
| Version         | `GrumPHP 1.5.0`
| Bug?            | no
| New feature?    | no
| Question?       | yes
| Documentation?  | no

Hello, I would like install PSR4 but, I can't find the installer. Is possible to install ? Thanks .

```
grumphp:
  tasks:
    phpcs:
      standard: [ "PSR12", "PSR1", "PSR4" ]
    phplint: ~

**Comment by veewee** [comment]
Hello,

This is a question you should ask on phpcs. We just pass the standards to the tool.
You can list all the standards installed by running `phpcs -i` as mentioned in the docs.

**Reply by GeoffreyOlivier** [reply]
Ok thanks.
